django_project/ford3/models/requirement.py

```diff
@@ -1,13 +1,14 @@
 from django.db import models
 from ford3.models.qualification import Qualification
+from ford3.enums.saqa_qualification_level import SaqaQualificationLevel
 
 
 class Requirement(models.Model):
     qualification_id = models.ForeignKey(
         Qualification,
         on_delete=models.CASCADE)
 
-    id = models.IntegerField(
+    id = models.AutoField(
```

Comment:
I just realized we have an id field.
Why we need id field explicitly?
Wasn't this handled in the background by Django?